website/content/plugins/devices/nvidia.mdx

```diff
@@ -41,13 +42,13 @@ The `nvidia-gpu` device plugin exposes the following environment variables:
 
 Additional environment variables can be set by the task to influence the runtime
 environment. See [Nvidia's
```

Comment:
```suggestion
environment. Refer to [NVIDIA's
```

Not technically part of this PR, but please change "See" to "Refer to". We don't use "see" because we [avoid ableist langugage](https://github.com/hashicorp/tutorials/blob/main/STYLE_GUIDE.md#avoid-ableist-language)